client/core/core.go

```diff
@@ -316,6 +312,8 @@ func (c *Core) tryCancel(dc *dexConnection, oid order.OrderID) (found bool, err
 	log.Infof("Cancel order %s targeting order %s at %s has been placed",
 		co.ID(), oid, dc.acct.host)
 
+	c.notify(newOrderNote("Cancelling order", "A cancel order has been submitted for order "+tracker.token(), db.Poke, tracker.coreOrderInternal()))
```

Comment:
This was the missing notification needed to get the order status column to update.